apps/bsd/src/app_root.tsx

```diff
@@ -96,6 +96,10 @@ export function AppRoot({ card, hardware }: AppRootProps): JSX.Element {
     adjudication: { remaining: 0, adjudicated: 0 },
     scanner: ScannerStatus.Unknown,
   });
+  const currentNumberOfBallots = useMemo(
```

Comment:
Why `useMemo` here?

Reply:
Well I realized I forget to add this to the dependency array for the zeroData useCallback so I'll fix that but since this (and currentUserType) are used as dependencies in the useCallback calls I am using useMemo so they don't get redefined (and therefore cause the useCallback function to get redefined) on every render. 

It's not added in this PR but I'm also planning to add a log which would log every time this variable changes, which would be a useEffect with both this and currentUserType in the dependency array.

Reply:
But isn't this (and the other case) using a primitive value? If it is the same primitive value it'll be `===` and memoizing won't do anything. Is an ESLint rule complaining about this?

Reply:
Ah that makes sense, I know I saw an eslint at some point in the past suggesting I use useMemo or something but I forget the exact situation and if it was a primitive value or not so then I just got in the habit of useMemo-ing things after that. I just tried removing this one and didn't see the eslint error.